apache-activemq-mixin/dashboards/apache-activemq-instance-overview.libsonnet

```diff
@@ -0,0 +1,1113 @@
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+        ],
+      },
+      unit: 's',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local garbageCollectionCountPanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(java_lang_g1_young_generation_collectioncount{' + matcher + ', instance=~"$instance"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{activemq_cluster}} - {{instance}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Garbage collection count / $__interval',
+  description: 'The recent increase in the number of garbage collection events for the JVM.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        axisShow: false,
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 25,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local alertsRow = {
+  datasource: promDatasource,
+  targets: [],
+  type: 'row',
+  title: 'Alerts',
+  collapsed: false,
+};
+
+local activemqAlertsPanel = {
+  datasource: promDatasource,
+  targets: [],
+  type: 'alertlist',
+  title: 'ActiveMQ Alerts',
+  description: 'Alerts for Apache ActiveMQ environment.',
+  options: {
+    alertInstanceLabelFilter: '',
+    alertName: '',
+    dashboardAlerts: false,
+    folder: {
+      title: 'Integrations - ActiveMQ',
+      uid: 'ac912ae2-f603-4a05-878f-42033c5b96f3',
+    },
```

Comment:
Thats a too specific, imho. Can go wrong easily.  I suggest to use alertInstanceLabelFilter filled in with what you have on your dashboard variables on top:
```
{job=~"[[job:regex]]", activemq_cluster=~"[[activemq_cluster:regex]]", instance=~"[[instance:regex]]"}
```
Note alternative [syntax](https://grafana.com/docs/grafana/latest/dashboards/variables/variable-syntax/#regex), as $var doesn't work in Alerts panel.